posts/hybrid-reference-profiles/index.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Creating hybrid reference profiles for InSituType"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-08-21"
+date-modified: "2024-08-21"
+categories: [cell typing]
+draft: false
+image: figures/heatmap_tile.png
+---
+
+
+## Background
+
+The InSituType cell typing algorithm relies on a "reference matrix" to perform supervised or semi-supervised cell typing.
+A reference matrix gives the expected gene expression profile of each cell type in a tissue;
+these are usually derived from previous scRNA-seq or spatial transcriptomics experiments. 
+
+Fairly often, it's convenient to create a hybrid reference matrix from two studies. 
+In a typical example, you may want to cell type a solid tissue with autoimmune disease. 
+There is likely a good scRNA-seq dataset available for the healthy cell types in your tissue, 
+but this dataset probably has poor coverage of immune cells, as immune cells are rare in 
+non-inflamed tissues, and most single cell datasets only sample some tens of thousands of cells.
+
+
+
+## Example
+
+Here's an example workflow for merging the HCA liver cell profiles with the 
```

Comment:
It would be good to include a reference to the HCA liver cell profiles, in part because I'm not sure that the CellProfileLibrary is well known. Something along the lines of this:
`. . . merging the HCA liver cell profiles (original data found [here](https://explore.data.humancellatlas.org/projects/4d6f6c96-2a83-43d8-8fe1-0f53bffd4674){target="_blank"} and pre-computed expression profiles found in NanoString's [CellProfileLibrary](https://github.com/Nanostring-Biostats/CellProfileLibrary/tree/master){target="_blank"}) with the . . . `

Again, double check the liver HCA link since I'm not sure if you're intending to show liver or colon.